packages/rum-vue/src/index.js

```diff
@@ -56,7 +56,10 @@ export const ApmVuePlugin = {
     /**
      * Provide the APM instance via $apm to be accessed in all Vue Components
      */
-    if (majorVersion >= 3) app.config.globalProperties.$apm = apm
+    if (majorVersion >= 3) {
+      app.config.globalProperties.$apm = apm
+      app.provide("$apm", apm)
+    }
```

Comment:
@Lightwight for some reason the PR is not showing clearly that lint is failing. If you execute ```npm run lint``` you will see the needed modifications you need to do on that piece of code

Reply:
@devcorpio Fixed the lint errors. Now it hangs on elasticsearch-ci/docs. Maybe retrigger?

Reply:
@Lightwight e2e added 👍 , once the build finishes I will merge your PR. thank you again for the time you invested on this

Reply:
@devcorpio @vigneshshanmugam Guys, thank you very much for your work and time 👍